src/Umbraco.Cms.Integrations.Commerce.Shopify/Client/src/property-editor/manifests.ts

```diff
@@ -0,0 +1,47 @@
+import { ManifestTypes, type ManifestPropertyEditorUi } from "@umbraco-cms/backoffice/extension-registry";
+import { manifests as amountManifest } from "../config/amount/manifests.js";
+import { manifests as authorizationManifest } from "../config/authorization/manifests.js";
+
+export const propertyEditorUiManifest : ManifestPropertyEditorUi = {
+    type: "propertyEditorUi",
+    alias: "Shopify.PropertyEditorUi.ProductPicker",
+    name: "Shopify Product Picker Property Editor UI",
+    element: () => import("./shopify-product-picker-property-editor.element.js"),
+    meta: {
+        label: "Shopify Product Picker",
+        icon: "icon-shopping-basket-alt",
+        group: "pickers",
+		propertyEditorSchemaAlias: 'Umbraco.Cms.Integrations.Commerce.Shopify.ProductPicker',
+        settings:{
+            properties:[
+                {
+                    alias: 'authorization',
+					label: 'Authorization',
+					description: 'Authorize your Shopify connection.',
+					propertyEditorUiAlias: 'Shopify.PropertyEditorUi.Authorization',
+                },
+                {
+                    alias: 'amount',
+					label: 'Amount',
+					description: 'Set a required range of items selected.',
+					propertyEditorUiAlias: 'Shopify.PropertyEditorUi.Amount',
+                    config: [
```

Comment:
This is the config for the required range of amount for the selected product list. The min and max values here are successfully displayed in the setting of the Shopify Product Picker data type, but I can't pass these to the property editor.

Reply:
Another approach I can think of is to create 2 new settings for Shopify Product Picker: Min and Max values. With that, I won't have to create a brand new property editor like current approach, just 2 simple `Umb.PropertyEditorUi.Integer`. It would be much simpler (I think!)